repose-aggregator/tests/performance-tests/test_vars/filters/scripting/javascript.yml

```diff
@@ -0,0 +1,29 @@
+{
+  dir_prefix: "filters/scripting",
+  script_lang: "javascript",
+  cloud: {
+    naming_prefix: "perf-filters-scripting-{{ script_lang }}"
```

Comment:
Although it does lead to a little extra boiler plate around the test, but a required file will never be forgotten. Or at least caught fairly early as opposed to forgetting to add it to a list in some other file.